layouts/partials/footer.html

```diff
@@ -1,15 +1,19 @@
 <footer class="row middle-xs center-xs">
 
+  {{ if .Site.Params.facebook }}
+    <div class="col-xs-3 col-md-2"><a target="_blank" href="https://facebook.com/{{ .Site.Params.facebook }}">GitHub</a></div>
```

Comment:
Hey @gustavotero7 thank you for the pull request! Would you mind fixing the link to say `Facebook` instead of `GitHub`, and adding the new param to `README.md` and `exampleSite/config.toml` before I merge it in?